app/src/main/java/com/canvearth/canvearth/MapsActivity.java

```diff
@@ -90,9 +86,9 @@ private void enableMyLocation() {
         }
     }
 
-    private void showToast(Context context, String text){
+    private void showToast(Context context, String text) {
         Toast toast = Toast.makeText(context, text, Toast.LENGTH_LONG);
-        toast.setGravity(Gravity.CENTER_HORIZONTAL|Gravity.CENTER_VERTICAL|Gravity.BOTTOM,
+        toast.setGravity(Gravity.CENTER_HORIZONTAL | Gravity.CENTER_VERTICAL | Gravity.BOTTOM,
```

Comment:
I think you can use `CENTER` instead of `CENTER_HORIZONTAL | CENTER_VERTICAL`.
It looks much better simpler to see, but it is just my personal preference. Never mind.

Reply:
Will do